pkg/agent/multicast/mcast_discovery.go

```diff
@@ -138,6 +142,28 @@ func (s *IGMPSnooper) validate(event *mcastGroupEvent) (bool, error) {
 	return true, nil
 }
 
+func (s *IGMPSnooper) addToIGMPReportNPStatsMap(item types.McastNPValidationItem, packetLen uint64) {
+	uuid := item.Uuid
```

Comment:
Not your code, but "Uuid should be "UUID" according to our coding convention.

Reply:
@liu4480